mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```diff
@@ -66,8 +67,8 @@ storage:
           #!/bin/bash
           set -euxo
```

Comment:
Not new but just noticed this is missing `pipefail`.

Reply:
will add it in next upload